src/schemas/Program/index.js

```diff
@@ -339,19 +325,18 @@ const resolvers = {
 			);
 
 			return hasPrivateField
-				? resolveSingleProgram(egoToken, shortName)
-				: resolveHTTPProgram(shortName);
+				? resolvePrivateSingleProgram(egoToken, shortName)
+				: resolvePublicSingleProgram(shortName);
 		},
```

Comment:
updating the names to exclude "http", because they are all http now. And include private or public.